Java2020/src/homeworkHierarchyAndComparable/Employee.java

```diff
@@ -0,0 +1,37 @@
+public abstract class Employee implements Comparable<Employee> {
+    String name;
+    Integer yearsOfExperience;
+
+
+    @Override
+    public int compareTo(final Employee o) {
+        int years = yearsOfExperience.compareTo(o.yearsOfExperience);
+        if (years == 0) {
+            return yearsOfExperience - o.yearsOfExperience;
+        }
+
+        return years;
+    }
```

Comment:
This implementation actually does nothing additional if to look at Integer.compareTo.
You could just return yearsOfExperience.compareTo(o.yearsOfExperience).
Think correct please and and more complex comparison. 
For example, sort employees by the length of their names.
If the lengths are equal, then compare by yearsOfExperience.

Reply:
Ok! Will fix it